src/pages/settings/Security/TwoFactorAuth/StepWrapper/StepWrapper.tsx

```diff
@@ -41,6 +41,7 @@ function StepWrapper({
         <ScreenWrapper
             shouldShowOfflineIndicator={false}
             shouldEnableKeyboardAvoidingView={shouldEnableKeyboardAvoidingView}
+            shouldEnableMaxHeight
```

Comment:
This to fix on iOS mWeb.

<img width="411" alt="Screenshot 2024-09-11 at 22 15 00" src="https://github.com/user-attachments/assets/7e7f5567-b5d6-4d1a-a4da-0d1182edc4c7">